database/migrations/2022_10_19_012446_create_publishers_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('publishers', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->char('publisherCode', 3);
+            $table->timestamps(); // Do we need timestamps?
```

Comment:
I don't think we need timestamps per the criteria but it could be a useful tool to track updates, etc. to the tables... Maybe this should be a group question?

Reply:
timestamps never hurt and can always come in handy, i would leave them